featurex/extractors/google.py

```diff
@@ -1,49 +1,56 @@
 from featurex.extractors.image import ImageExtractor
 from featurex.stimuli.image import ImageStim
 from featurex.google import GoogleVisionAPITransformer
-from featurex import Value, Event
+from featurex.extractors import ExtractorResult
 import numpy as np
 
 
 class GoogleVisionAPIExtractor(GoogleVisionAPITransformer, ImageExtractor):
 
     def _extract(self, stim):
         if isinstance(stim, ImageStim):
-            is_image = True
             stim = [stim]
-        else:
-            is_image = False
+
         request =  self._build_request(stim)
         responses = self._query_api(request)
 
-        events = []
+        features = []
+        data = []
         for i, response in enumerate(responses):
             if response:
                 annotations = response[self.response_object]
-                values = self._parse_annotations(stim[i], annotations)
-                onset = stim[i].onset if hasattr(stim[i], 'onset') else i
-                ev = Event(onset=onset, duration=stim[i].duration, values=values)
-                events.append(ev)
-            else:
-                events.append(Event())
+                feat, values = self._parse_annotations(annotations)
+                features += feat
+                data += values
 
-        if is_image:
-            return events[0].values
-        return events
+        data = [data]
+        onsets = [stim[i].onset if hasattr(stim[i], 'onset') else i for i in range(len(responses))]
+        durations = [stim[i].duration for i in range(len(responses))]
+        return ExtractorResult(data, stim, self, features=features, 
+                                onsets=onsets, durations=durations)
 
 
 class GoogleVisionAPIFaceExtractor(GoogleVisionAPIExtractor):
 
     request_type = 'FACE_DETECTION'
     response_object = 'faceAnnotations'
-
-    def _parse_annotations(self, stim, annotations):
+    likelihood_dict = {'UNKNOWN': 0.0, 
```

Comment:
Are these official values from Google? If not, I think we should probably stick with the strings... As much as I dislike the qualitative categories the API returns here, I think we want to avoid transforming values, as the user might not realize what's going on. A reasonable compromise would be to add a `convert_likelihood`  argument to the `transform` call that defaults to `False`, but does the conversion if the user explicitly sets `True`.